frontend/public/components/nav/NavHeader.scss

```diff
@@ -19,6 +19,9 @@
     margin-right: var(--pf-global--spacer--sm);
   }
 
+  &__dropdown-toggle {
+    cursor: default !important;
+  }
```

Comment:
This doesn't work for me and still shows `cursor: hand`


https://user-images.githubusercontent.com/139310/193224833-271505bc-1e32-4265-aa44-613e17b26a8c.mp4

Reply:
Fixed it